helm-charts/splunk-otel-collector/templates/_helpers.tpl

```diff
@@ -324,16 +324,46 @@ compatibility with the old config group name: "otelAgent".
 {{- end }}
 {{- end -}}
 
+{{/*
+The name of the gateway service.
+*/}}
+{{- define "splunk-otel-collector.gatewayServiceName" -}}
+{{  (include "splunk-otel-collector.fullname" . ) | trunc 63 | trimSuffix "-" }}
+{{- end -}}
```

Comment:
Is this helper being used?

Reply:
Yes, it's being used in the reducer-deployment.yml - it's the destination for the reducer's output.

Reply:
Actually - I intended to use it there, but I ended up just using fullname.  Let me fix that up real quick.